lib/api.js

```diff
@@ -17,7 +17,12 @@ module.exports = (robot) => {
     }
 
     const github = GitHubAPI({
+      Octokit: ProbotOctokit,
       logger: req.log,
+      retry: {
+        // disable retries to test error states
+        enabled: false,
```

Comment:
We think that some assumptions in the app will break if we turn on automatic retries, so leaving them disabled for now :)